DaemonRenderer/Source/Src/Windowing/Window.cpp

```diff
@@ -427,6 +432,51 @@ DAEbool Window::IsValid() const noexcept
     return m_handle != nullptr;
 }
 
+#ifdef DAEMON_OS_WINDOWS
+
+HWND Window::GetWin32Window() const noexcept
+{
+    return glfwGetWin32Window(m_handle);
+}
+
+#endif
+
+#pragma endregion
+
 #pragma endregion
 
+#pragma region Operators
+
+Window& Window::operator=(Window&& in_move) noexcept
+{
+    m_logger               = in_move.m_logger;
```

Comment:
Both the logger and the handle can be moved too here since they are pointers, the actual objects will remain untouched

Reply:
A pointer is trivially copyable so moving it is less optimized.